webpack.config.js

```diff
@@ -40,8 +46,12 @@ module.exports = {
     ]
   },
 
-  plugins: [
+  plugins: PROD ? [
     new CommonsChunkPlugin({ name: "angular2", filename: "angular2.js", minChunks: Infinity }),
-    new CommonsChunkPlugin({ name: "common", filename: "common.js" })
+    new UglifyJsPlugin({
+      compress: { warnings: false }
+    })
+  ] : [
+    new CommonsChunkPlugin({ name: "angular2", filename: "angular2.js", minChunks: Infinity })
```

Comment:
I don't really like the repetition here, maybe compile the plugins array at the top and just push on plugins based on the environment.